site/content/entry/citra-progress-report-2017-july.md

```diff
@@ -20,33 +20,33 @@ What's nice about this is that it can be used as a quick and easy way for 3DS de
 <br />
 <!--
 title_id = 0004000000174600
-commit_hash = ????
+commit_hash = a40e60c4bc7ab9c559ad6e3f1769b1128d68c8d5
 -->
-<i>How jagged</i>
+*How jagged*
 </p>
 
-Fortunately (and to much rejoice) wwylele stepped into the ring to wrestle with this issue. They prepared [test programs](https://github.com/wwylele/ctrhwtest/tree/master/texture-copy-test) to help gain an understanding of how hardware handles a TextureCopy in comparison to Citra and implemented an accurately functioning TextureCopy mode for Citra's emulation of the 3DS GPU.
+Fortunately (and to much rejoice) wwylele stepped into the ring to wrestle with this issue. They prepared a [test program](https://github.com/wwylele/ctrhwtest/tree/master/texture-copy-test) to help gain an understanding of how hardware handles the TextureCopy operation in comparison to Citra.
 
 <p style="text-align: center; font-size: small; padding: 1%">
 <img style="padding: 0% 0% 1% 0%" height="50%" width="50%" alt="Pokemon Super Mystery Dungeon During Deoxy's and Rayquaza's Face Off (IN S P A C E)" src="/images/entry/citra-progress-report-2017-july/texturecopy-after.png" />
 <br />
 <!--
 title_id = 0004000000174600
-commit_hash = 686fde7e526e024716baa3aa3ba887d1a2479d41
+commit_hash = 57438cf51f3bdae71a694d2ebe96ad563968781c
 -->
-Now we get an accurate portrayal of deoxys having a hard time
+*Now we get an accurate portrayal of deoxys having a hard time*
```

Comment:
Why is this italicized? I thought you had italicized line 25 because the image was also slanted (which would've been a clever visual joke). And if you're going for consistency, the blurb on line 51 isn't italicized. 😅